### PR TITLE
[DOC] correct some grammar in comments in config.default.js

### PR DESCRIPTION
**Pull request by rubo77**
## Description
This corrects some grammar errors in comments in config.default.js

- [x] My code follows the code style of this project. (CI will test it anyway and also needs approval)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by genofire — APPROVED** [review]

**Review by xf- — CHANGES_REQUESTED** [review]
prefix is a prefix....

**Comment by rubo77** [comment]
So better like this?

value can be a node attribute (depth 1) or any of the functions from lib/utils/node.js having the prefix "show"

File: config.default.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ module.exports = function () {
     'fullscreen': true,
     'fullscreenFrame': true,
     'nodeAttr': [
-      // value can be a node attribute (1 depth) or a a function in utils/node with prefix show
+      // value can be a node attribute (depth 1) or any of the functions with a name starting with "show" from lib/utils/node.js with
```
Comment (xf-) [inline config.default.js]: of cource the functions are in lib and ending is unimportant (not both not needed in a require)

**Review by xf- — CHANGES_REQUESTED** [review]

File: config.default.js (hunk 1 above)
Comment (rubo77) [reply config.default.js]: Der alte Kommentar war zwar eine korrekte Gedankenstütze, wenn man schon wusste, was zu tun war, aber ich hoffe, der neue Kommentar erklärt einem neuen User, der dabei ist, die config.default.js an seine Community anzupassen, jetzt deutlich, wie sich das `value` zusammensetzten muss, bzw. wo man die möglichen Werte dafür findet

**Comment by xf-** [comment]
This code will be removed in next released

**Pull request closed without merging**